lib/nlp/nlp-manager.js

```diff
@@ -447,7 +463,19 @@ class NlpManager {
       classifier.features = classifierClone.features;
       const lrc = classifier.settings.classifier;
       const { logistic } = classifierClone;
-      lrc.observations = logistic.observations;
+
+      lrc.observations = {};
+      Object.entries(logistic.observations).forEach(([label, matrix]) => {
+        lrc.observations[label] = [];
+        matrix.forEach((row, rowIndex) => {
+          // Create a array filled with as many zeros as features
+          const features = Array(classifier.features.length).fill(0);
+          // Set the features for the positions stored with 1. The others remains as zero.
+          row.forEach(featPosition => features.splice(featPosition, 0, 1));
```

Comment:
features is an array... why not
```javascript
row.forEach(featPosition => features[featPosition] = 1);
```
splice is process consuming, while direct assignment only replace the value in the memory.

Reply:
you are totally right. I don't know why I were using splice 😰 

I commited the change on 7f10b3b